.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+package-lock.json
```

Comment:
Unrelated: we shouldn't check in `package-lock.json` for this repo, because we want the developer to always get the latest version of the SDK when they build it— not a pinned version.